src/Scrolling.md

```diff
@@ -1,19 +1,19 @@
 
 # LCD Position and Scrolling
 
-These registers can be accessed even during Mode 3, but they have no
-effect until the end of the current scanline.
+These registers can be accessed even during Mode 3, but they may not take
+effect immediately.
```

Comment:
I don't think the change was an improvement. Immediatly can mean anything in this context while the previous formulation had more information about until when they can (not) show any effect.

Reply:
The previous formulation was wrong, actually. The scroll registers are re-read on each tile fetch, except for the low 3 bits of `SCX`, which are only read at the beginning of the scanline (during the initial shifting).

I figured that explaining this nuance (which also opens the can of worms of "what is a tile fetch?") here would be detrimental to comprehension.

Reply:
Got it and I agree with you. Maybe put an info box with the tile fetch thing? It's interesting.

Reply:
Or a link to the FIFO article?

Reply:
I prefer adding a brief explanation like the one you gave me here and link to the FIFO article

Reply:
Footnote it is, then.

Reply:
I instead went and turned the "mid-frame" tip box into a section, where I also added this info.